test/configCases/loaders/issue-3359/index.js

```diff
@@ -0,0 +1,22 @@
+"use strict";
+
+// 1. Test some-loader!some-loader
+// 3. Test json-loader?query1!json-loader
+// 4. Test some-loader?query1!some-loader with query object from webpack.config
+
+it("should fail when a loader is called twice", () => {
+	// eslint-disable-next-line node/no-unpublished-require, node/no-missing-require
+	require("some-loader!./a");
+	// FIXME how I catch this error?
+	// error.message.should.be.exact("hey, you're applying a loader multiple times, don't do that man!");
```

Comment:
Why not for this configCases we set in the config `bail: true` ( this may already be turned on by default but lets check anyways)